src/views/ContinuationIn/ContinuationInBusinessHome.vue

```diff
@@ -1,46 +1,58 @@
 <template>
   <div id="continuation-in-business-home">
-    <v-fade-transition>
-      <div
-        v-show="showSpinner"
-        class="loading-container grayed-out"
-      >
-        <div class="loading__content">
-          <v-progress-circular
-            color="primary"
-            size="50"
-            indeterminate
-          />
-          <div class="loading-msg white--text">
-            Fetching data
-          </div>
-        </div>
-      </div>
-    </v-fade-transition>
-
     <!-- Existing Business Information -->
     <section class="mt-10">
       <header id="existing-Business-information">
         <h2>Existing Business Information</h2>
         <p class="mt-4">
-          Enter information about your existing business.
+          Enter information about your existing business. If your company is extra-provinvicially
+          registered in B.C., that registration will be made historic when this continuation in
+          application is processed.
         </p>
       </header>
 
-      <div>**TODO: Add component</div>
+      <v-card
+        flat
+        class="py-8 px-6"
+        :class="{ 'invalid-section': getShowErrors && !existingBusinessInformationValid }"
+      >
+        <v-expand-transition>
+          <ExtraproRegistration
+            v-if="!manualBusinessInfoActive"
+            @active="extraproRegistrationActive = $event"
+            @valid="extraproRegistrationValid = $event"
+          />
+        </v-expand-transition>
+
+        <div :class="{ 'mt-6': !extraproRegistrationActive && !manualBusinessInfoActive }" />
+
+        <v-expand-transition>
+          <ManualBusinessInfo
+            v-if="!extraproRegistrationActive"
+            @active="manualBusinessInfoActive = $event"
+            @valid="manualBusinessInfoValid = $event"
+          />
+        </v-expand-transition>
+      </v-card>
```

Comment:
The above is a good example of this component knowing as little as possible about the sub-components. And the sub-components don't need to know about each other.